common/djangoapps/student/models.py

```diff
@@ -2879,3 +2879,14 @@ class LogoutViewConfiguration(ConfigurationModel):
     def __unicode__(self):
         """Unicode representation of the instance. """
         return u'Logout view configuration: {enabled}'.format(enabled=self.enabled)
+
+
+class AccountRecovery(models.Model):
+    """
+    Model for storing information for user's account recovery in case of access loss.
+    """
+    user = models.OneToOneField(User, related_name='account_recovery', on_delete=models.CASCADE)
+    secondary_email = models.EmailField(verbose_name=_('Secondary email address'), unique=True, null=False, blank=False)
```

Comment:
I think we can also add a help text for secondary email like:

help_text=_('Secondary email address to recover linked edX account.')

Reply:
Done. However, I have updated the text to 
```
help_text=_('Secondary email address to recover linked account.')
```
Since this change will be a part of edx-platform and will probably be used by people other than edX.